emergency-protocol/scripts/deploy_accounts.ts

```diff
@@ -0,0 +1,27 @@
+import fs from 'fs'
```

Comment:
Why are these functions defined as scripts?

You can use Starknet CLI or Gauntlet CLI to deploy an account for manual testing:

- https://docs.starknet.io/docs/CLI/commands#starknet-deploy_account
- https://github.com/smartcontractkit/chainlink-starknet/tree/develop/packages-ts/starknet-gauntlet-oz